### DISCUSSION TITLE
Installing

### DISCUSSION THREAD

**Discussion by NANEDERLAND** [anchovie91471/12-Step-Meeting-List-Xtras#10] [General]
Hi,

First of all, thanks for putting so much time and energy in this extension. We are planning to install this plugin. We just cannot figure out how. Uploading the zip doesnt work, overwriting files doesnt work and uploading the folder to the plugin ftp doesnt work.

Could you guide us maybe how to install this?

Kind regards,
NA Netherlands

**Comment by anchovie91471** [comment]
Sounds like you may have some folder permissions issues. Have you tried clicking “add new” from the plug-ins page and searching for “12 step meeting list” and installing it via the Wordpress plug-ins directory?

**Reply by NANEDERLAND** [reply]
thank you for your quick repsons. The meeting list plugin is installed, we just want to add '12-Step-Meeting-List-Xtras'. But I dont know how...

**Reply by anchovie91471** [reply]
Lol! My apologies! I missed which plug-in you were posting from and assumed 12 Step Meeting List”

when you say “doesn’t work” do you mean the plugin shows up in Wordpress but won’t install or it doesn’t show up at all under plug-ins? 

You do have to have 12 step meeting list installed and activated before you can activate Xtras from the plug-ins page.

Send me any errors and a bit more detail on your problem and I’ll be happy to troubleshoot!

**Comment by anchovie91471** [comment]
The install process should be to download the zip from GitHub and unzip into your wp-content/plugins folder; then activate it from the plugins page in Wordpress as you would any other plugin. I overlooked install instructions in my readme. Sorry.

If you’re doing this and it isn’t working, there is definitely a problem I need to address.

**Comment by anchovie91471** [comment]
I just tried installing from the zip and saw some errors. I fixed those and pushed up a new version. Feel free to grab a new zip and try again.

**Comment by NANEDERLAND** [comment]
Amazing, it works! What a great and fast response, thank you so much!

**Comment by anchovie91471** [comment]
Anytime! Feel free to send me any feedback you ought have. I’m excited to see what the community thinks and if it’s helpful.

**Comment by seanalansnyder** [comment]
@anchovie91471 , You may want to update the Installation section of the Readme.md.
https://github.com/code4recovery/12-Step-Meeting-List-Xtras#installation

It looks like it was copy-pasted from the original 12-Step-Meeting-List page, and the URL within that Installation section still references the 12-Step-Meeting-List plugin page, NOT 12-Step-Meeting-List-xtras.

**Reply by anchovie91471** [reply]
Yikes! Good catch! Once we are out of beta we will have our own page, but definitely need better instructions on the meantime. Thanks!!
